Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+
+**Default HoloLens (1st gen) profile**
+
+We have added a new profile for HoloLens (1st gen) development that includes some of the
+recommended MRTK configurations for best performance.
+
+To configure your application for HoloLens (1st gen) optimized settings, set the
+Mixed Reality Toolkit's **Active Profile** to **DefaultHoloLens1ConfigurationProfile**.
+
+![Default HoloLens (1st gen) Configuration Profile](./Images/ReleaseNotes/DefaultHoloLens1ConfigurationProfile.png)
+
+**Dependency Window**
+
+A [Dependency Window](Tools/DependencyWindow.md) has been added which displays how assets reference and depend on each other.
+This tool can be used to easily determine which assets within a project are not being used.
+
+**Extension Service Creation Wizard**
+
+Making the transition from singletons to services can be difficult. The [Extension Service Creation Wizard](Tools/ExtensionServiceCreationWizard.md)
+supplements documentation and sample code by enabling devs to create new services easily
+
+**Fingertip Cursor**
+
+The fingertip cursor has been updated to better match the HoloLens 2 shell.
+
+**Hand Menu Graduated from Experimental**
+
+The HandConstraint and HandConstraintPalmUp solvers are now an official feature (no longer experimental) and have improved documentation.
+
+The HandConstraintPalmUp solver now has a toggle to enforce the hand’s fingers are coplanar before activating.
+
+**HoloLens 2 Button**
+
+- Improved many visual details to match the HoloLens 2 shell including
+    - Compressing visuals
+    - Far interaction support
+    - Focus highlight
+    - Shader effects
+- HoloLens 2 style Round Button has been added
+
+**HoloLens 2 Shell Parity**
+
+This release updates the MRTK to better mirror the featires. behaviors and visuals of the HoloLens 2 shell experience. This [GitHub](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4200) issue describes the changes.
+
+**IMixedRealityRaycastProvider**
+
+The Input System was modified to add a reference to an IMixedRealityRaycastProvider.
+
+Specify your desired raycast provider in the Input System's configuration profile. 
+
+![Selecting the Raycast provider](Images/ReleaseNotes/SelectRaycastProvider.png)
+
+**Input Animation Recording**
+ 
+MRTK 2.0.0 features a [recording system](InputSimulation/InputAnimationRecording.md) by which head movement and hand tracking
+data can be stored in animation files. The recorded data can then be played back using the [input simulation system](InputSimulation/InputSimulationService.md).
+
+**Launch applications from within your Unity app**
+
+On HoloLens 2 and Windows Mixed Reality Immersive applications, you can now launch applications
+from within your Uniity application. See the HandInteractionExamples scene in the Examples package
+(Demos\HandInteraction\Scenes\HandInteractionDemos.unity) for a demonstration of using the LaunchUri script
+to start an external application.
```

Comment:
This isn't an MRTK feature - we have an example that shows it but it seems a bit disingenuous to state this.

Reply:
reworked